README.md

```diff
@@ -155,8 +155,13 @@ Example:
 `CDNOriginHost` is the domain name of the _origin_ server (i.e. the server running Cloak) under `CDN` mode. This only
 has effect when `Transport` is set to `CDN`. If unset, it will default to the remote hostname supplied via the
 commandline argument (in standalone mode), or by Shadowsocks (in plugin mode). After a TLS session is established with
-the CDN server, this domain name will be used in the HTTP request to ask the CDN server to establish a WebSocket
-connection with this host.
+the CDN server, this domain name will be used in the `Host` header of the HTTP request to ask the CDN server to
+establish a WebSocket connection with this host.
+
+`CDNRequestUrlPath` is the url path used to build websocket request sent under `CDN` mode, and also only has effect
+when `Transport` is set to `CDN`. If unset, it will default to "/". This option is used to build the first line of the
```

Comment:
It actually defaults to the empty string, which is the current behaviour we should keep. "ws://foo.com/" and "ws://foo.com" can be treated differently by web servers

Reply:
HTTP requires non-empty path, and the websocket client of go will actually send a request starting with `GET / HTTP/1.1` if `u.Path==""`. But it might brings confusion to code readers. What about adding comments for the field in `RawConfig`?

Reply:
Hmmm I need to check the behaviour of `gorilla/websocket`, since we're currently passing in host:port without the trailing slash

Reply:
Ok you're right that the trailing slash immediately after the domain name doesn't matter - that is `ws://foo.com/` and `ws://foo.com` are always the same. Saying it defaults to `/` is fine here, but maybe explicitly set it to `/` too in `RawConfig` if the string is empty? This doesn't affect behaviour but just makes the code correspond to documentation better.